tests/lib/saneresponse.js

```diff
@@ -0,0 +1,21 @@
+// A common function to confirm that the response sent by the server meets our standards.
+"use strict";
+var fluid  = require("infusion");
+var jqUnit = require("node-jqunit");
+var gpii   = fluid.registerNamespace("gpii");
+
+require("../lib/superset");
+
+fluid.registerNamespace("gpii.pouch.lucene.tests");
+gpii.pouch.lucene.tests.isSaneResponse = function (response, body, statusCode, expected) {
```

Comment:
There is still some unique wiring, for the common checks I deferred to the comparable function in {{gpii-express}}.